.github/workflows/ci-build-windows.yaml

```diff
@@ -27,8 +27,11 @@ jobs:
       
       - name: Format
         run: |
-          $res = (gofmt -s -l .) -replace "$_"
+          go install golang.org/x/tools/cmd/goimports@latest
+          $goimp = (Join-path -Path (go env GOPATH) -ChildPath "\bin\goimports")
```

Comment:
Have you verified that "\bin\goimports" works on Windows?

Reply:
Yes. It works on windows and the test builds are also working fine.